homeassistant/components/life360/strings.json

```diff
@@ -1,27 +1,27 @@
 {
-  "config": {
-    "step": {
-      "user": {
-        "title": "Life360 Account Info",
-        "data": {
-          "username": "[%key:common::config_flow::data::username%]",
-          "password": "[%key:common::config_flow::data::password%]"
+    "config": {
```

Comment:
Can we keep it two space indented?

Reply:
Yeah vscode auto formatted this, will go back to two spaces.